sunbeam-python/sunbeam/jobs/manifest.py

```diff
@@ -74,18 +77,77 @@ class Manifest:
 
     @classmethod
     def load(cls, manifest_file: Path) -> "Manifest":
+        """Load the manifest with the provided file input"""
         with manifest_file.open() as file:
             return Manifest(**yaml.safe_load(file))
 
     @classmethod
     def load_latest_from_clusterdb(cls) -> "Manifest":
+        """Load the latest manifest from clusterdb"""
         try:
             manifest_latest = clusterClient().cluster.get_latest_manifest()
             return Manifest(**yaml.safe_load(manifest_latest.get("data")))
         except Exception as e:
             LOG.debug(f"Got error in creating latest manifest object: {str(e)}")
             return Manifest()
 
+    @classmethod
+    def load_on_default(cls, manifest_file: Path) -> "Manifest":
+        """Load manifest and override the default manifest"""
+        with manifest_file.open() as file:
+            override = yaml.safe_load(file)
+            default = cls.get_default_manifest_as_dict()
+            utils.merge_dict(default, override)
+            return Manifest(**default)
+
+    @classmethod
+    def load_latest_from_clusterdb_on_default(cls) -> "Manifest":
+        """Load the latest manifest from clusterdb"""
+        default = cls.get_default_manifest_as_dict()
+        try:
+            manifest_latest = clusterClient().cluster.get_latest_manifest()
+            override = yaml.safe_load(manifest_latest.get("data"))
+            utils.merge_dict(default, override)
+        except Exception as e:
+            LOG.debug(f"Got error in creating latest manifest object: {str(e)}")
```

Comment:
Shouldn't that method stop if it failed to create the manifest ?

Reply:
Done